packages/transaction-controller/src/TransactionController.ts

```diff
@@ -269,8 +269,6 @@ export class TransactionController extends BaseControllerV1<
 
   private readonly pendingTransactionTracker: PendingTransactionTracker;
 
-  private readonly cancelMultiplier: number;
-
   private readonly speedUpMultiplier: number;
```

Comment:
I appreciate the scope of the issue is just for the cancel multiplier, but the higher level goal is to remove unnecessary divergence and configuration in the clients.

Is it worth removing `speeUpMultiplier` at the same time since I don't believe it was ever used. I only added it out of consistency with cancel.

Reply:
Removed it as well, leaving the both global constants separated still.